afs/models.py

```diff
@@ -155,35 +261,120 @@ def switch_repo(self, repo_name=None):
+                raise ValueError('Model not found.')
+                return None
+
+    def get_model_info(self, model_name, model_repository_name=None):
+        """Get model info, including created_at, tags, evaluation_result. (V2 API)
+        
+        :param model_name: model name
+        :param model_repository_name: The name of model repository.
+        :return: dict model info
+        """
+        if self.api_version == 'v2':
+            if not self.get_model_repo_id(model_repository_name=model_repository_name):
+                raise ValueError('Model_repository not found.')
+
+            model_id = self.get_model_id(model_name=model_name, model_repository_name=model_repository_name)
+
+            if model_id:
+                extra_paths = [self.repo_id, self.sub_entity_uri, model_id]
+                resp = self._get(extra_paths=extra_paths)
+            else:
+                raise ValueError('Model not found.')
+        else:
+            raise NotImplementedError('v1 API is not support this method.')
+            return None
         return resp.json()
 
 
-    def _create(self, data, files=None, extra_paths=[]):
+    def delete_model_repository(self, model_repository_name):
+        """Delete model repository.
+        
+        :param model_repository_name: model repository name.
+        :return: bool 
+        """
+        if self.api_version == 'v2':
+            if not self.get_model_repo_id(model_repository_name):
+                raise ValueError('Model_repository not found.')
+                return False
+            extra_paths = [self.repo_id]
+            resp = self._del(extra_paths=extra_paths)
+
+        else:
+            raise NotImplementedError('v1 API is not support this method.')
+            return None
+
+        if int(resp.status_code / 100) == 2:
+            return True
+        else:
+            return False
+
+
+    def delete_model(self, model_name, model_repository_name=None):
+        """Delete model.
+        
+        :param model_name: model name.
+        :param model_repository_name: model repository name.
+        :return: bool
+        """
+        if self.api_version == 'v2':
+            if not self.get_model_repo_id(model_repository_name=model_repository_name):
+                raise ValueError('Model_repository not found.')
+                return False
+            model_id = self.get_model_id(model_name, model_repository_name=model_repository_name, last_one=False)
+            if not model_id:
+                raise ValueError('Model not found.')
+            extra_paths = [self.repo_id, self.sub_entity_uri, model_id]
+            resp = self._del(extra_paths=extra_paths)
+        else:
+            raise NotImplementedError('v1 API is not support this method.')
+            return None
+
+        if int(resp.status_code / 100) == 2:
+            return True
+        else:
+            return False
+
+
+    def _create(self, data, files=None, extra_paths=[], form='json'):
```

Comment:
Why not use `*args` and `**kwargs` to pass all argument into `requests`? I think `_create()` only need to manipulate the URL and other arguments just pass to `requests` would be easier to maintain.

Reply:
OK, open #26 to follow up.